package.json

```
@@ -66,23 +72,24 @@
     "all-contributors-cli": "^6.24.0",
     "auto": "^10.42.2",
     "esbuild": "^0.16.14",
-    "esbuild-plugin-text-replace": "^1.2.0",
-    "eslint": "^8.31.0",
+    "eslint": "^8.52.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jest": "^27.2.0",
     "jest": "^29.3.1",
     "jest-environment-node": "^29.3.1",
     "ovsx": "^0.7.1",
-    "prettier": "^2.8.1",
+    "prettier": "^3.1.0",
+    "prettier-eslint": "^16.1.2",
     "ts-node": "^10.9.1",
     "typescript": "^4.9.5",
     "vsce": "^2.15.0"
   },
   "dependencies": {
     "find-up": "^6.3.0",
     "ignore": "^5.2.4",
-    "prettier-eslint": "^15.0.1"
+    "require-relative": "^0.8.7",
+    "synckit": "^0.8.6"
```

```suggestion
    "synckit": "^0.9.0"
```

It's much smaller now.

---

See #207 @idahogurl